src/lcm/input_processing/util.py

```diff
@@ -88,33 +97,52 @@ def get_variable_info(model: Model) -> pd.DataFrame:
     return info.loc[order]
 
 
-def _get_auxiliary_variables(
-    state_variables: list[str],
+def _get_variables_that_enter_concurrent_valuation(
+    states_and_actions_names: list[str],
     function_info: pd.DataFrame,
     user_functions: dict[str, UserFunction],
 ) -> list[str]:
-    """Get state variables that only occur in next functions.
+    """Get variables that enter the concurrent valuation.
```

Comment:
```suggestion
    """Get states and action variables that enter the concurrent valuation.
```

Should be symmetric to function below.